dashed/assets/javascripts/modules/dashed.js

```diff
@@ -169,16 +169,52 @@ var px = (function () {
         }
         return qrystr;
       },
+      widgetHeader: function () {
```

Comment:
IMO `getWidgetHeader()` is more readable. also would this fail (return `null` / `undefined`) for explorer view? it's not in a `<li>` there?